spec.html

```diff
@@ -21438,6 +21743,31 @@ <h1>Runtime Semantics: ClassFieldDefinitionEvaluation</h1>
       </emu-note>
     </emu-clause>
 
+    <emu-clause id="sec-runtime-semantics-classstaticblockdefinitionevaluation" type="sdo" aoid="ClassStaticBlockDefinitionEvaluation">
+      <h1>Runtime Semantics: ClassStaticBlockDefinitionEvaluation</h1>
+      <p>With parameter _homeObject_.</p>
+      <emu-grammar>ClassStaticBlock: `static` `{` ClassStaticBlockBody `}`</emu-grammar>
+      <emu-alg>
+        1. Let _lex_ be the running execution context's LexicalEnvironment.
+        1. Let _privateScope_ be the running execution context's PrivateEnvironment.
+        1. Let _sourceText_ be the empty sequence of Unicode code points.
+        1. Let _bodyFunction_ be OrdinaryFunctionCreate(%Function.prototype%, _sourceText_, &laquo; &raquo;, |ClassStaticBlockBody|, ~non-lexical-this~, _lex_, _privateScope_).
+        1. Perform MakeMethod(_bodyFunction_, _homeObject_).
+        1. Return the ClassStaticBlockDefinition Record { [[BodyFunction]]: _bodyFunction_ }.
+      </emu-alg>
+      <emu-note>The function _bodyFunction_ is never directly accessible to ECMAScript code.</emu-note>
+    </emu-clause>
+
+    <emu-clause id="sec-runtime-semantics-evaluateclassstaticblockbody" type="sdo" aoid="EvaluateClassStaticBlockBody">
+      <h1>Runtime Semantics: EvaluateClassStaticBlockBody</h1>
+      <p>With parameter _functionObject_.</p>
+      <emu-grammar>ClassStaticBlockBody : ClassStaticBlockStatementList</emu-grammar>
+      <emu-alg>
+        1. Perform ? FunctionDeclarationInstantiation(_functionObject_, &laquo; &raquo;).
+        1. Return the result of evaluating |ClassStaticBlockStatementList|.
```

Comment:
Evaluation of `ClassStaticBlockStatementList : StatementList` can chain to `StatementList`, but
Evaluation of `ClassStaticBlockStatementList : [empty]` needs to be defined.

Reply:
I've added this to sec-class-definitions-runtime-semantics-evaluation, thanks.